runelite-client/src/main/java/net/runelite/client/plugins/performance/PerformanceOverlay.java

```diff
@@ -45,30 +46,35 @@
  * This locks "FPS:" into one position (the far top right corner of the canvas),
  * along with a locked position for the FPS value.
  */
-public class FpsOverlay extends Overlay
+public class PerformanceOverlay extends Overlay
 {
 	private static final int MAX_FPS = 50;
 	private static final int FPS_SIZE = MAX_FPS + 1;
 	private static final int Y_OFFSET = 14;
-	private static final int VALUE_X_OFFSET = 15;
+
 	private static final String FPS_STRING = " FPS";
 
 	// Cache of FPS number strings from 00-50
 	private final String[] fpsNums;
 
 	// Local dependencies
-	private final FpsConfig config;
+	private final PerformanceConfig config;
 	private final Client client;
 
+	@Inject
+	private PerformancePlugin plugin;
+
 	// Often changing values
 	private boolean isFocused = true;
+	private int VALUE_Y_AXIS_PING;
```

Comment:
Why are these 2 (VALUE_Y.., point) instance variables and not local in the render method?

Reply:
Changes made